src/services/__tests__/redisSessionStorage.test.ts

```diff
@@ -412,9 +426,9 @@ describe("RedisSessionStorage#set", () => {
     redisMethodImplFromError(mockSetEx, "OK");
     // FIMS Token
     redisMethodImplFromError(mockSetEx, "OK");
-    redisMethodImplFromError(mockSetEx, "OK");
-    redisMethodImplFromError(mockSadd, 1);
-    redisMethodImplFromError(mockSmembers, []);
+    // redisMethodImplFromError(mockSetEx, "OK");
+    // redisMethodImplFromError(mockSadd, 1);
+    // redisMethodImplFromError(mockSmembers, []);
```

Comment:
They were causing other tests to fail, because of refactoring. Removed in [6b9207c](https://github.com/pagopa/io-backend/pull/1029/commits/6b9207cfefa8a52445e8164602eb7b3a0519171f), thanks